[2024-04-24T21:50:28.204+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-24T21:50:28.224+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_pipe.extract manual__2024-04-24T21:50:25.883178+00:00 [queued]>
[2024-04-24T21:50:28.346+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_pipe.extract manual__2024-04-24T21:50:25.883178+00:00 [queued]>
[2024-04-24T21:50:28.346+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-24T21:50:28.354+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): extract> on 2024-04-24 21:50:25.883178+00:00
[2024-04-24T21:50:28.364+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=294) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-24T21:50:28.363+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_pipe', 'extract', 'manual__2024-04-24T21:50:25.883178+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/air_quality/test.py', '--cfg-path', '/tmp/tmp0j_s9wva']
[2024-04-24T21:50:28.365+0000] {standard_task_runner.py:63} INFO - Started process 296 to run task
[2024-04-24T21:50:28.365+0000] {standard_task_runner.py:91} INFO - Job 93: Subtask extract
[2024-04-24T21:50:28.401+0000] {task_command.py:426} INFO - Running <TaskInstance: test_pipe.extract manual__2024-04-24T21:50:25.883178+00:00 [running]> on host 7dc195e25eb1
[2024-04-24T21:50:28.467+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_pipe' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2024-04-24T21:50:25.883178+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-24T21:50:25.883178+00:00'
[2024-04-24T21:50:28.467+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-24T21:50:28.478+0000] {logging_mixin.py:188} INFO - 2024-04-24
[2024-04-24T21:50:28.478+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-24T21:50:28.478+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/air_quality/tasks.py", line 59, in test
    print({{"ts"}})
          ^^^^^^^^
TypeError: unhashable type: 'set'
[2024-04-24T21:50:28.485+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=test_pipe, task_id=extract, execution_date=20240424T215025, start_date=20240424T215028, end_date=20240424T215028
[2024-04-24T21:50:28.491+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 93 for task extract (unhashable type: 'set'; 296)
[2024-04-24T21:50:28.537+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-04-24T21:50:28.550+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-24T21:50:28.552+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
